test/controllers/project_skill_controller_test.exs

```diff
@@ -83,13 +80,13 @@ defmodule CodeCorps.ProjectSkillControllerTest do
       id = json["data"]["id"] |> String.to_integer
       project_skill = ProjectSkill |> Repo.get(id)
 
-      assert json["data"]["id"] == "#{project_skill.id}"
-      assert json["data"]["type"] == "project-skill"
+      json
+      |> Map.get("data")
+      |> assert_result_id(project_skill.id)
+      |> assert_jsonapi_relationship("project", project.id)
+      |> assert_jsonapi_relationship("skill", skill.id)
+
       assert json["data"]["attributes"] == %{}
```

Comment:
`|> assert_attributes_equals(%{})` might not be overkill and is just as explicit. Thoughts?